src/main/java/io/strimzi/kafka/bridge/SinkBridgeEndpoint.java

```diff
@@ -494,7 +494,7 @@ private void commitOffsets(boolean clear) {
                     new io.vertx.kafka.client.consumer.OffsetAndMetadata(offsetAndMetadata.offset() + 1, offsetAndMetadata.metadata()));
         });
 
-        if (offsets != null && !offsets.isEmpty()) {
+        if (!offsets.isEmpty()) {
```

Comment:
Is it impossible for offsets to be null?

Reply:
If you take a look at both implementations of `getOffsets()`, but are returning instancied objects. So `offsets` cannot be null.